client/components/chart/legend.jsx

```diff
@@ -44,10 +44,19 @@ const Legend = React.createClass( {
 
 	propTypes: {
 		activeTab: React.PropTypes.object.isRequired,
-		tabs: React.PropTypes.array.isRequired,
-		activeCharts: React.PropTypes.array.isRequired,
-		availableCharts: React.PropTypes.array.isRequired,
-		clickHandler: React.PropTypes.func.isRequired
+		tabs: React.PropTypes.array,
+		activeCharts: React.PropTypes.array,
+		availableCharts: React.PropTypes.array,
+		clickHandler: React.PropTypes.func
```

Comment:
Good point. I'm taking care of all the janitorial work to the legend in https://github.com/Automattic/wp-calypso/pull/14495. I will apply this comment there.